External/FEXCore/include/FEXCore/Core/SignalDelegator.h

```diff
@@ -96,14 +96,14 @@ namespace Core {
 
   private:
     struct HostSignalHandler {
-      FEXCore::HostSignalDelegatorFunction Handler{};
+      std::vector<FEXCore::HostSignalDelegatorFunction> Handlers{};
       FEXCore::HostSignalDelegatorFunction FrontendHandler{};
     };
     std::array<HostSignalHandler, MAX_SIGNALS + 1> HostHandlers{};
 
   protected:
     void SetHostSignalHandler(int Signal, HostSignalDelegatorFunction Func, bool Required) {
-      HostHandlers[Signal].Handler = std::move(Func);
+      HostHandlers[Signal].Handlers.emplace_back(std::move(Func));
```

Comment:
FYI `emplace_back(std::move(...))` is a bit of an anti-pattern: It does not improve performance since there's already an overload for `push_back` that takes rvalues, but `emplace_back` produces unreadable error messages when something goes wrong. (Similarly, using `emplace_back` does not compensate for an accidentally missing `std::move`!)

See https://quuxplusone.github.io/blog/2021/03/03/push-back-emplace-back/#when-all-else-is-equal-prefer-push_back-to-emplace_back for details if you're interested :)

Reply:
Resolved.